### PR TITLE
Updated orders case statement for fixed amount discounts

### PR DESCRIPTION
**Pull request by fivetran-avinash**
## PR Overview
**This PR will address the following Issue/Feature:** #77 

**This PR will result in the following new package version:**

This will merge into a v.12.0 branch that @fivetran-catfritz is working on.

**Please provide the finalized CHANGELOG entry which details the relevant changes included in this PR:**

## 🪲 Bug Fixes 🪛
- Corrected the `fixed_amount_discount_amount` logic to appropriately bring in fixed amount discounts in `shopify__orders`. [PR #78](https://github.com/fivetran/dbt_shopify/pull/78)
- Removed the `index=1` filter in `stg_shopify__order_discount_code` in the `dbt_shopify_source` package to ensure all discount codes are brought in for every orders. For customers with multiple discount codes in an order, this could update the `count_discount_codes_applied` field in the `shopify__orders` and `shopify__daily_shop` models. [PR #80](https://github.com/fivetran/dbt_shopify_source/pull/80)

## 🔧 Under the Hood 🔩
- Updated the pull request templates. [PR #78](https://github.com/fivetran/dbt_shopify/pull/78)

### Basic Validation
Please acknowledge that you have successfully performed the following commands locally:
- [x] dbt run –full-refresh && dbt test
- [NA] dbt run (if incremental models are present)

Before marking this PR as "ready for review" the following have been applied:
- [x] The appropriate issue has been linked, tagged, and properly assigned
- [x] All necessary documentation and version upgrades have been applied
- [Catherine will regenerate before release] docs were regenerated (unless this PR does not include any code or yml updates)
- [x] BuildKite integration tests are passing
- [x] Detailed validation steps have been provided below

### Detailed Validation
Please share any and all of your validation steps:

Seeded and ran the updated files from `integration_tests` with `fixed_amount` to make sure that `fixed_amount_discount_amount` was bringing in the right values, and ensure that `count_discount_codes_applied` was also correct per `order_id`.

### CODE REVIEW

File: .DS_Store
Comment (fivetran-joemarkiewicz) [inline .DS_Store:1]: Can you remove this file and add it to the `.gitignore`

File: integration_tests/.DS_Store
Comment (fivetran-joemarkiewicz) [inline integration_tests/.DS_Store:1]: Can you remove this file and add it to the `.gitignore`.

File: packages.yml [hunk 1]
```diff
@@ -1,3 +1,4 @@
 packages:
-- package: fivetran/shopify_source
-  version: [">=0.11.0", "<0.12.0"]
\ No newline at end of file
+  - git: https://github.com/fivetran/dbt_shopify_source.git
+    revision: bugfix/remove-order-discount-code-filter
+    warn-unpinned: false
```
Comment (fivetran-joemarkiewicz) [inline packages.yml:4]: Reminder to swap before release

**Review by fivetran-joemarkiewicz — APPROVED** [review]
@fivetran-avinash this looks good to go! Please just be sure to remove the .DS_Store files before merging to the release branch.

File: packages.yml (hunk 1 above)
Comment (fivetran-avinash) [reply packages.yml]: swapped to the release branch

**Pull request merged**